README.md

```diff
@@ -58,3 +58,4 @@ print(Animal.alive)
 ```
 Hint: Use magic method.
 
+> Check your code using this [checklist](checklist.md) before pushing your solution.
```

Comment:
let's make it noticeable 
`### Note: Check your code using this [checklist]([checklist.md](http://checklist.md/)) before pushing your solution.`